style/components/navbar.css

```diff
@@ -0,0 +1,29 @@
+.menu {
```

Comment:
The component's class name should be the same as the file name – to better find it.